lib/src/rules/public_member_api_docs.dart

```diff
@@ -78,6 +78,10 @@ class PublicMemberApiDocs extends LintRule implements NodeLintRule {
   @override
   void registerNodeProcessors(
       NodeLintRegistry registry, LinterContext context) {
+    if (!isInLibDir(context.currentUnit.unit, context.package)) {
```

Comment:
@scheglov : is there ever a chance that `context.currentUnit` would be null?  (Or the corresponding  `unit` be null?)  Should I be null-checking anywhere  here?

Reply:
No, `context.currentUnit` cannot be `null`. And its `unit` also cannot be `null`.

Reply:
thank you!